src/core/Attachment.hpp

```diff
@@ -0,0 +1,33 @@
+#ifndef ATTACHMENT_HPP
```

Comment:
Use #pragma once

Reply:
Any way to let clang-tidy programmatically convert the portable way to the pragma?

Reply:
I don't know... In QtCreator you can set #pragma once by default when new file is created